clab/clab.go

```diff
@@ -73,10 +65,7 @@ func WithRuntime(name string, d bool, dur time.Duration, gracefulShutdown bool)
 		if rInit, ok := runtime.ContainerRuntimes[name]; ok {
 			r := rInit()
 			err := r.Init(
-				runtime.WithConfig(&runtime.RuntimeConfig{
-					Timeout: dur,
-					Debug:   d,
-				}),
+				runtime.WithConfig(rtconfig),
```

Comment:
We can discuss if we should use a copy of the runtimeConfig provided to avoid any race conditions.
On the other hand one could use that handle to change runtime config by intention.
Unsure myself.